histoDB/histodb-dataserver/pom.xml

```diff
@@ -137,6 +137,7 @@
         <dependency>
             <groupId>eu.itesla_project</groupId>
             <artifactId>cim1-import</artifactId>
+            <version>0.1-SNAPSHOT</version>
```

Comment:
I didn't notice that histoDB was not a child of ipst parent pom.xml. Is there a reason?